src/main/resources/META-INF/plugin.xml

```diff
@@ -227,6 +212,27 @@
             <!-- In $default keymap `control` key (Windows/Linux) corresponds to `cmd` key on macOS -->
             <keyboard-shortcut keymap="$default" first-keystroke="control alt shift M" />
         </action>
+
+        <action id="GitMachete.NextBranchCheckoutAction"
+                class="com.virtuslab.gitmachete.frontend.actions.traverse.NextBranchCheckoutAction">
+            <add-to-group group-id="GitRepositoryActions" anchor="last"/>
+            <keyboard-shortcut keymap="$default" first-keystroke="control alt shift T" second-keystroke="control shift L" />
```

Comment:
I'd make the second stroke more consistent... what are the limitations actually? are you free to use anything for the second stroke, or are there sequence that are gonna be captured as _first_ stroke anyway? 🤔 like, if you use `F1` for second stroke, will it be captured as second stroke, or first stroke of single-stroke F1 shortcut (Help, by default)?

Reply:
The only limitation is that, the second stroke should not overlap with any existing key shortcut. such as ctrl+C

Reply:
with `G` it was not getting triggered

Reply:
Aww okay I see now hmmm

Reply:
Okay so assuming the second stroke overlaps with an existing shortcut... what specifically happens? The existing single-stroke shortcut takes precedence (and e.g. Copy action is triggered instead of our double-stroke action)?

Reply:
> Copy action is triggered on Ctrl+C instead of our action

Yes.